CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.1.2] - 2021-02-17
```

Comment:
The date will need an update depending on when you are ready to release

Reply:
you want to put today's date here then ?

Reply:
yep, I was waiting until your answer so I commit the two changes at the same time